### PR TITLE
Added code caitlin write for DHS

### PR DESCRIPTION
**Pull request by mholiv**

### CODE REVIEW

File: dns_test.yml [hunk 1]
```diff
@@ -0,0 +1,15 @@
+---
+- name: Configre DNS Server
+  hosts: all
+  become: True
+
+  vars:
+    #dns_domain:
+    dns_nameservers: ['8.8.8.8', '8.8.4.4']
```
Comment (defionscode) [inline dns_test.yml]: make the `dns_nameservers` expression like this 
```
vars:
    dns_nameservers:
        - '8.8.8.8'
        - '8.8.4.4'
```
This is simply the preferred style.

File: ldap_test.yml [hunk 2]
```diff
@@ -0,0 +1,103 @@
+---
+- name: configure ldap
+  hosts: all
+  become: True
```
Comment (defionscode) [inline ldap_test.yml]: Doesnt need to change but for future reference yes/Yes/True/true/No/no are all valid boolean expressions in yaml

File: dns_test.yml (hunk 1 above)
Comment (mholiv) [inline dns_test.yml]: Fixed

---
Caitlin
e. caitlin@cissols.com<mailto:caitlin@cissols.com>

On Mar 7, 2016, at 2:31 PM, Jonathan Davila <notifications@github.com<mailto:notifications@github.com>> wrote:


In dns_test.yml<https://github.com/defionscode/vmware_x/pull/1#discussion_r55257705>:

> @@ -0,0 +1,15 @@
> +---
> +- name: Configre DNS Server
> +  hosts: all
> +  become: True
> +
> +  vars:
> +    #dns_domain:
> +    dns_nameservers: ['8.8.8.8', '8.8.4.4']


make the dns_nameservers expression like this

vars:
    dns_nameservers:
        - '8.8.8.8'
        - '8.8.4.4'


This is simply the preferred style.

-

File: module_test.yml [hunk 3]
```diff
@@ -0,0 +1,82 @@
...
+
+
+- name: configure Spacewalk client
+  hosts: testservers
+  sudo: True
+  vars:
+    spacewalk_server: INSERT-SPACEWALK-SERVER-HOSTNAME-HERE
+    activation_key: false
+  tasks:
+    - name: rhel activation key
```
Comment (defionscode) [inline module_test.yml]: add a line break between `tasks` and `- name`

File: module_test.yml [hunk 4]
```diff
@@ -0,0 +1,82 @@
...
+      set_fact:
+        activation_key: 1-rhel-{{ ansible_distribution_major_version }}-x86_64-server-key
+        distro_shortname: rhel
+      when: ansible_distribution == "Red Hat Enterprise Linux"
+    - name: centos activation key
```
Comment (defionscode) [inline module_test.yml]: Add one line break between tasks

File: module_test.yml [hunk 5]
```diff
@@ -0,0 +1,82 @@
...
+        activation_key: 1-centos-{{ ansible_distribution_major_version }}-x86_64-server-key
+        distro_shortname: centos
+      when: ansible_distribution == "CentOS"
+
+    - name: remove spacewalk cert chain
+      yum:
+        name: rhn-org-trusted-ssl-cert
+        state: removed
+
+      name: install cert chain
```
Comment (defionscode) [inline module_test.yml]: missing hyphen before name

File: module_test.yml [hunk 6]
```diff
@@ -0,0 +1,82 @@
...
+    - name: centos activation key
+      set_fact:
+        activation_key: 1-centos-{{ ansible_distribution_major_version }}-x86_64-server-key
+        distro_shortname: centos
+      when: ansible_distribution == "CentOS"
+
+    - name: remove spacewalk cert chain
+      yum:
+        name: rhn-org-trusted-ssl-cert
+        state: removed
```
Comment (defionscode) [inline module_test.yml]: state should be `absent` instead of `removed`

File: module_test.yml [hunk 7]
```diff
@@ -0,0 +1,82 @@
...
+  tasks:
+    - name: rhel activation key
+      set_fact:
+        activation_key: 1-rhel-{{ ansible_distribution_major_version }}-x86_64-server-key
+        distro_shortname: rhel
+      when: ansible_distribution == "Red Hat Enterprise Linux"
+    - name: centos activation key
+      set_fact:
+        activation_key: 1-centos-{{ ansible_distribution_major_version }}-x86_64-server-key
+        distro_shortname: centos
```
Comment (defionscode) [inline module_test.yml]: static variables should be defined in inventory in the `vars` declaration, not via set_fact

File: spacewalk_test.yml [hunk 8]
```diff
@@ -0,0 +1,34 @@
...
+  - name: remove spacewalk cert chain
+    yum:
+      name: rhn-org-trusted-ssl-cert
+      state: removed
+
+  - name: install cert chain
+    shell: "rpm -Uvh http://{{ spacewalk_server }}/pub/rhn-org-trusted-ssl-cert-1.0-2.noarch.rpm"
+
+  - name: register spacewalk client
+    shell: "rhnreg_ks --force --serverUrl=https://{{ spacewalk_server }}/XMLRPC --activationkey={{ activation_key }}"
```
Comment (defionscode) [inline spacewalk_test.yml]: use `command`

File: library/vmware_add_hdd.py [hunk 9]
```diff
@@ -0,0 +1,170 @@
+#!/usr/bin/python
+import ssl
```
Comment (defionscode) [inline library/vmware_add_hdd.py]: looks like you import ssl but don't actually use it anywhere

Comment (mholiv) [inline library/vmware_add_hdd.py]: Updated.

---
Caitlin
e. caitlin@cissols.com<mailto:caitlin@cissols.com>

On Mar 8, 2016, at 10:04 PM, Jonathan Davila <notifications@github.com<mailto:notifications@github.com>> wrote:


In library/vmware_add_hdd.py<https://github.com/defionscode/vmware_x/pull/1#discussion_r55466790>:

> @@ -0,0 +1,170 @@
> +#!/usr/bin/python
> +import ssl


looks like you import ssl but don't actually use it anywhere

-

**Pull request merged**